Classical models (#89)
mlsa-book/MLSA: book/P3C13_classical.qmd

In contrast to the boosting methods discussed in @sec-boost, model-based boosting is a generic process for fitting iterative models, however it is unlikely a generic boosting pipeline will outperform the specific algorithms introduced in @sec-boost.

#### Stacking {.unlisted .unnumbered}

When stacking is used, multiple models are fit on the same training data, and their predictions are used as training data to a meta-model.
The meta-model uses the model predictions to predict the target from the original training data.
Fitting a meta-model, often a simple linear model, results in fitted coefficients that weight the input models according to how close or far they were from the truth.
Effectively this results in a weighted average of predictions when a linear meta-model is used, where the weights are determined by model accuracy.

## Conclusion

:::: {.callout-warning icon=false}

## Key takeaways

* Classical statistical models broadly fall into three categories: non-parametric estimators, semi-parametric models that include a baseline estimate, and fully-parametric estimates. Each has its own advantages and disadvantages.

> * Non-parametric estimators are used throughout survival analysis with the Kaplan-Meier estimator being a very common component in many machine learning models.

Author: Random forests, GBMs in the losses, the SVM that uses restricted survival, ...

Maybe instead

```suggestion
* Non-parametric estimators are used throughout survival analysis, often as components in more complex machine learning models.
```
